.github/workflows/applications.yml

```diff
@@ -18,15 +18,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ['9.6']
-        cabal: ['3.10']
-        os: ['ubuntu-20.04', 'ubuntu-22.04', 'macos-latest', 'macos-14']
+        ghc: ['9.6.6', '9.8.2', '9.10.1']
+        cabal: ['3.12']
+        os: ['ubuntu-20.04', 'ubuntu-22.04', 'macos-14']
         cabalcache: ['true']
         flags: ['+build-tool']
```

Comment:
The build with `+build-tool` still has a bunch of `-wx-partial` warnings.

Reply:
@edmundnoble sorry, I missed your comments when I hit the merge button (from the GitHub phone app). I'll address your comments in a follow up PR.